Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -45,17 +47,15 @@ void Update()
         } 
     }
 
-    void NotifyChasingNurse()
+    private void OnCollisionEnter(Collision other)
     {
-        //TODO: Add a parameter to this function, layer mask
-        Collider[] hitColliders = Physics.OverlapSphere(transform.position, loudEffectRadius);
-        foreach (Collider collider in hitColliders)
-        {
-            // if collider is the chasing nurse 
-            // {
-            // call it's function 
-            // }
-        }
+        if (!hasBeenBumped || hasHit || other.gameObject.CompareTag("Player")) return;
+        
+        Vector3 hitPoint = other.GetContact(0).point;
+        Vector3 hitNormal = other.GetContact(0).normal;
+
+        gameObject.GetComponent<NoisePing>().SpawnNoisePing(hitPoint, hitNormal);

```

Comment:
I think just to keep it clean, it would be nice to do save the reference to `gameObject.GetComponent<NoisePing>()` on Start.